config: set names automatically on schema upgrade (#9187)
tarantool/tarantool: changelogs/unreleased/config-automatic-names.md

## feature/config

> * If Tarantool is congired via yaml file or etcd, then
>   instance/replicaset/cluster names will be automatically set on schema upgrade.

Reviewer: ```suggestion
* If Tarantool is congired via a YAML file or etcd, then
  instance/replicaset/cluster names are automatically set on schema upgrade.
```
